app/browser/reducers/bookmarkFoldersReducer.js

```diff
@@ -75,10 +66,8 @@ const bookmarkFoldersReducer = (state, action, immutableAction) => {
           action.get('moveIntoParent')
         )
 
-        if (syncUtil.syncEnabled()) {
-          const destinationDetail = state.getIn(['sites', action.get('destinationKey')])
-          state = syncUtil.updateSiteCache(state, destinationDetail)
-        }
+        const destinationDetail = state.getIn(['bookmarkFolders', action.get('destinationKey')])
```

Comment:
This way we have state path defined only in one place `bookmarkFoldersState` and if we want to refactor anything it will help us a lot.